### PR TITLE
Added helm chart optional containers

### PR DESCRIPTION
**Pull request by daiyyr**
Give an option for those who need auth add-ons such as keycloak

### CODE REVIEW

File: charts/helm-chart/kubernetes-dashboard/values.yaml [hunk 1]
```diff
@@ -355,3 +355,30 @@ serviceMonitor:
   labels: {}
   ## Here annotations can be added to the serviceMonitor
   annotations: {}
+
+## Optional containers for auth addons such as keycloak-gatekeeper
+optionalContainers:
+  ## Wether to enable dashboard-metrics-scraper
```
Comment (maciaszczykm) [inline charts/helm-chart/kubernetes-dashboard/values.yaml:361]: Looks like copy paste leftover.

File: charts/helm-chart/kubernetes-dashboard/values.yaml [hunk 2]
```diff
@@ -355,3 +355,30 @@ serviceMonitor:
...
+  #       cpu: 50m
+  #       memory: 50Mi
+  #     requests:
+  #       cpu: 50m
+  #       memory: 50Mi
+  #   terminationMessagePath: /dev/termination-log
+  #   terminationMessagePolicy: File
+  #   volumeMounts:
+  #     - name: keycloak-gatekeeper-secret
+  #       mountPath: /etc/secrets
```
Comment (maciaszczykm) [inline charts/helm-chart/kubernetes-dashboard/values.yaml:384]: Please remove it.

File: charts/helm-chart/kubernetes-dashboard/values.yaml (hunk 1 above)
Comment (daiyyr) [reply charts/helm-chart/kubernetes-dashboard/values.yaml]: It was an example of an optional container. Removed

File: charts/helm-chart/kubernetes-dashboard/values.yaml (hunk 2 above)
Comment (daiyyr) [reply charts/helm-chart/kubernetes-dashboard/values.yaml]: Removed

**Comment by daiyyr** [comment]
Hi @maciaszczykm, comments resolved, can you please have a look again. Thank you!

File: charts/helm-chart/kubernetes-dashboard/values.yaml [hunk 3]
```diff
@@ -355,3 +355,9 @@ serviceMonitor:
   labels: {}
   ## Here annotations can be added to the serviceMonitor
   annotations: {}
+
+## Optional containers for auth addons such as keycloak-gatekeeper
```
Comment (maciaszczykm) [inline charts/helm-chart/kubernetes-dashboard/values.yaml:359]: I'd avoid pointing to any specific addons. Let's just say auth addons here and we can merge. You'll also need to bump version.

Comment (daiyyr) [reply charts/helm-chart/kubernetes-dashboard/values.yaml]: Thank you for the advise. Updated

**Comment by maciaszczykm** [comment]
You didn't update the version.

**Comment by daiyyr** [comment]
> You didn't update the version.

Sorry was checking to rebase any dependency update but found none. Bumped the version to 6.0.3.

**Comment by daiyyr** [comment]
Fixed linting error. Please tag with lgtm again

**Comment by daiyyr** [comment]
Happy new year! When you come back from holidays, can you please review the change trying fix the linting error

**Pull request merged**